book/page_creation.rst

```diff
@@ -2,207 +2,197 @@
 
-.. code-block:: text
+    class LuckyController
+    {
+        /**
+         * @Route("/lucky/number")
+         */
+        public function numberAction()
+        {
+            $number = rand(0, 100);
 
-    http://localhost/app_dev.php/random/100
+            return new Response(
+                '<html><body>Lucky number: '.$number.'</body></html>'
+            );
+        }
+    }
 
-Actually, you'll be able to replace ``100`` with any other number to generate
-numbers up to that upper limit. To create the page, follow the simple two-step
-process.
+Before diving into this, test it out!
 
-.. note::
+.. code-block:: text
 
-    The tutorial assumes that you've already downloaded Symfony and configured
-    your webserver. The above URL assumes that ``localhost`` points to the
-    ``web`` directory of your new Symfony project. For detailed information
-    on this process, see the documentation on the web server you are using.
-    Here are some relevant documentation pages for the web server you might be using:
+    http://localhost:8000/app_dev.php/lucky/number
 
-    * For Apache HTTP Server, refer to `Apache's DirectoryIndex documentation`_
-    * For Nginx, refer to `Nginx HttpCoreModule location documentation`_
+.. tip::
 
-Before you begin: Create the Bundle
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    If you setup a proper virtual host in :doc:`Apache or Nginx </cookbook/configuration/web_server_configuration>`,
+    replace ``http://localhost:8000`` with your host name - like
+    ``http://symfony.dev/app_dev.php/lucky/number``.
 
-Before you begin, you'll need to create a *bundle*. In Symfony, a :term:`bundle`
-is like a plugin, except that all the code in your application will live
-inside a bundle.
+If you see a lucky number being printed back to you, congratulations! But
+before you run off to play the lottery, check out how this works.
 
-A bundle is nothing more than a directory that houses everything related
-to a specific feature, including PHP classes, configuration, and even stylesheets
-and JavaScript files (see :ref:`page-creation-bundles`).
+The ``@Route`` above ``numberAction()`` is called an *annotation* and it
+defines the URL pattern. You can also write routes in YAML (or other formats):
+read about this in the :doc:`routing </book/routing>` chapter. Actually, most
+routing examples in the docs have tabs that show you how each format looks.
 
-Depending on the way you installed Symfony, you may already have a bundle called
-AcmeDemoBundle. Browse the ``src/`` directory of your project and check
-if there is a ``DemoBundle/`` directory inside an ``Acme/`` directory. If those
-directories already exist, skip the rest of this section and go directly to
-create the route.
+The method below the annotation - ``numberAction`` - is called the *controller*
+and is where you build the page. The only rule is that a controller *must*
+return a Symfony :ref:`Response <component-http-foundation-response>` object
```

Comment:
not sure if liking to components would benefit anyone. I see lots of people getting confused about these docs